packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -60,16 +61,18 @@ export type SchemaDrivenAnnotationEditorProps = {
   schemaId?: string
   /** May be used to directly provide a JSON Schema to use for the form */
   validationSchema?: JSONSchema7
-  /** Optionally supply a ref to the form to handle submission externally with `formRef.current.submit()`. If provided, the editor will not render its own submit UI. */
+  /** Optionally supply a ref to the form to handle submission externally with `formRef.current.submit()`. */
   formRef?: React.RefObject<RJSF>
-  /** Provide live input validation. This can cause major performance degradation. */
+  /** Provide live input validation. This can cause performance degradation. By default, liveValidate will be true if an entity with a schema and existing annotations is being edited */
   liveValidate?: boolean
```

Comment:
The original issue pointed out how confusing it is to see that an entity has invalid annotations, but you can't actually see which ones are invalid until you click 'Save'!

We can make this more user-friendly by turning on `liveValidate`. Since we started using this library, the performance of `liveValidate` has gotten significantly better due to improvements in RJSF.

If this does become a performance bottleneck, we can revisit and consider other strategies.